modules/op-chains-signing-secrets-ecdsa.adoc

```diff
@@ -0,0 +1,53 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="chains-signing-secrets-ecdsa_{context}"]
+= Generating the x509 key pair
+
+You can generate the `x509` key pair by setting the `generateSigningSecret` parameter to `true`. 
```

Comment:
I think we should say it as a field instead of parameter

```suggestion
You can generate the `x509` key pair by setting the `generateSigningSecret` field to `true`. 
```